packages/drizzle/src/queries/sanitizeQueryValue.ts

```diff
@@ -100,10 +100,11 @@ export const sanitizeQueryValue = ({
         }
         idType = typeMap[mixedType]
       } else {
-        // LIMITATION: Only cast to the first relationTo id type,
-        // otherwise we need to make the db cast which is inefficient
```

Comment:
I left this note as a warning to us in the future. You are making a tradeoff on performance by casting the id column.